agent/host.go

```diff
@@ -134,5 +163,39 @@ func createRootNode(g *graph.Graph) (*graph.Node, error) {
 		m.SetField("VirtualizationRole", hostInfo.VirtualizationRole)
 	}
 
+	var lshwMap map[string]interface{}
+	lshw, err := exec.Command("lshw", "-quiet", "-json").Output()
+	if err == nil {
+		err = json.Unmarshal(lshw, &lshwMap)
+		if err == nil {
+			parseLshw(lshwMap)
+			m.SetField("Hardware", lshwMap)
```

Comment:
if you use:

```
m.SetFieldAndNormalize("Hardware", lshwMap)
```

you may be able to eliminate the need for calling parseLshw()

Reply:
I'm sorry I don't get why. I need to rename some children node. That's is very lshw specific.